arbitrum-docs/node-running/running-an-archive-node.mdx

```diff
@@ -56,15 +64,17 @@ An archival node is a full (nitro) node that also builds an archive of historica
 - You could run both Arbitrum One and classic in archive mode. Arbitrum One has been upgraded to Nitro, the latest Arbitrum tech stack; "Arbitrum Classic" is our term for the old, pre-Nitro tech stack. The Nitro node databases have the raw data of all blocks, including pre-Nitro blocks. However, Nitro nodes cannot execute anything on pre-Nitro blocks. Arbitrum Nova started as a Nitro chain, so has no classic blocks.
 - Here we provide examples for running both Arbitrum One and Arbitrum Classic in archival mode.
   
-- An example of how to run an Arbitrum One archival node:
-  ```shell
-  docker run --rm -it -v /some/local/dir/arbitrum:/home/user/.arbitrum -p 0.0.0.0:8547:8547 -p 0.0.0.0:8548:8548 offchainlabs/nitro-node:v2.0.11-8e786ec --l1.url https://l1-node:8545 --l2.chain-id= --http.api=net,web3,eth,debug --http.corsdomain=* --http.addr=0.0.0.0 --http.vhosts=* --node.caching.archive 
-  ```
 - An example of how to run an Arbitrum Classic archival node:
   ```shell
   docker run --rm -it -v /some/local/dir/arbitrum-mainnet/:/home/user/.arbitrum/mainnet -p 0.0.0.0:8547:8547 -p 0.0.0.0:8548:8548 offchainlabs/arb-node:v1.4.5-e97c1a4 --l1.url=https://l1-node:8545/ --node.chain-id=42161 --l2.disable-upstream --node.cache.allow-slow-lookup --core.checkpoint-gas-frequency=156250000
   ```
-- Note the above commands both map to 8547 port on their hosts, if you want to run those two at the same time, you should edit those mapping to different ports, and add `--node.rpc.classic-redirect=<classic node RPC>` to your Nitro node. In addition to make sure they can connect correctly, you can see [this docker guide](https://www.middlewareinventory.com/blog/docker-network-example/).
+
+- An example of how to run an Arbitrum One archival node:
```

Comment:
Should we change this from Arbitrum One to Arbitrum Nitro too?
Also, latest version is v2.0.13-174496c

Reply:
Yeah, I planed to do that when this pr get merged since other file also contain outdate image version.